types/react-responsive-masonry/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for react-responsive-masonry 2.1
+// Project: https://github.com/cedricdelpoux/react-responsive-masonry#readme
+// Definitions by: themashcodee <https://github.com/themashcodee>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { CSSProperties, ReactNode, ReactElement } from 'react';
+
+export interface MasonryProps {
+    children: ReactNode;
```

Comment:
```suggestion
    children: ReactNode | ReactNode[];
```
https://github.com/cedricdelpoux/react-responsive-masonry/blob/master/src/Masonry/index.js#L61